### PR TITLE
reeeeee

### PR DESCRIPTION
**Pull request by ghost**
memez

### CODE REVIEW

**Comment by DinduStuffin** [comment]
+1 :^)

**Comment by doobiedoo23** [comment]
+1

File: hippiestation/code/modules/mob/living/say.dm [hunk 1]
```diff
@@ -12,3 +12,10 @@
 		return FALSE
 		
 	return TRUE
+
+/mob/living/say(message, bubble_type,var/list/spans = list(), sanitize = TRUE, datum/language/language = null)
+	if(findtext(message, "fart man"))
+		to_chat(src, "<span class='boldwarning'>Fucking die.</span>")
+		gib()
+		playsound_local(get_turf(src), 'hippiestation/sound/misc/slidewhistle_down.ogg', 100, FALSE, pressure_affected = FALSE)
```
Comment (KayeArray) [inline hippiestation/code/modules/mob/living/say.dm:20]: you should be calling this before gib()

**Comment by DinduStuffin** [comment]
why the hell didn't we merge this
this would be amazing

**Pull request closed without merging**